lib/models/updateAction.js

```diff
@@ -26,25 +26,35 @@ var util = require('util'),
     config = require('../../config'),
     constants = require('../constants'),
     myutils = require('../myutils'),
-    serviceField = 'ev__service',
-    tenantField = 'ev__tenant',
     errors = {};
 
 function doIt(action, event, callback) {
-    /*jshint camelcase: false */
     var service,
         tenant,
         headers,
-        updateOrion = {
+        updateOrion,
+        id, type, name, value, isPattern;
```

Comment:
Not sure in which point of the code, but remember adding `attrType` to the list of parameters that may use a template.